.github/label-actions.yml

```diff
@@ -1,9 +1,34 @@
 # Configuration for Github App - https://github.com/dessant/label-actions
+#
+# Note: Be aware of the edge cases of YAML when writing multiline strings:
+#   - https://yaml-multiline.info/
+#   - https://github.com/dessant/label-actions/issues/1
 pulls:
   actions:
     needs-docs:
       comment: |
         Thanks for your pull request, before this can be merged - corresponding documentation for your module is required:
+
           - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
           - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
-          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
\ No newline at end of file
+          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
+
+issues:
+  actions:
+    termux:
+      comment: |
+        Termux is not officially supported. https://github.com/rapid7/metasploit-framework/issues/11023
+
+        However, Metasploit reportedly does work with Termux.
+
+        Refer to the following for more information:
+
+        * https://wiki.termux.com/wiki/Metasploit_Framework
+        * termux/termux-packages/issues/715
```

Comment:
Behind the scenes I've also added a new [termux](https://github.com/rapid7/metasploit-framework/labels/termux) label. 

I have decided _not_ to close issues in this scenario, as sometimes issues are labeled as termux - but left open, and sometimes they are also closed.

Original message courtesy of @bcoles 

Example:

![image](https://user-images.githubusercontent.com/60357436/78929626-a7fab880-7a9a-11ea-81aa-4f22a8c5397b.png)